docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,185 @@
+---
+slug: quickstart
+append_help_link: true
+title: Scan your first project with Semgrep
+hide_title: false
+description: Learn how to set up Semgrep and scan your first project
+tags:
+  - quickstart
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Learn how to set up Semgrep, scan your first project for security issues, and view your findings.
+
+:::info prerequisites
+You must have Python 3.7 or later installed on the machine where the Semgrep CI is running.
+:::
+
+1. Navigate to [Semgrep](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub** or **Sign in with GitLab**. Follow the on-screen prompts to grant Semgrep the needed permissions and proceed.
+
+2. Provide the **Organization display name** you'd like to use, then click **Create new organization**.
+
+3. Launch your CLI, and follow the instructions on the **Scan a project on your machine** page. For your convenience, we've reproduced that information here and added instructions for Windows users.
```

Comment:
```suggestion
3. Launch your CLI, and follow the instructions on the **Scan a project on your machine** page. The following section reproduces that information and adds instructions for Windows users.
```
- Is the "Scan a project on your machine" page supposed to be a link?
- From the [google style guide:](https://developers.google.com/style/word-list#we) "...It's OK to use we to refer to the organization that's represented as the author of the document as long as the antecedent is clear." - we'd have to identify who "we" are in a document, so we writers do it sparingly.

Whole bit about it: [Second and first person](https://developers.google.com/style/person)